system-int-test/test/cmr/system_int_test/ingest/provider_ingest_permissions_test.clj

```diff
@@ -3,7 +3,7 @@
   (:require
    [clojure.string :as string]
    [clojure.test :refer :all]
-   [cmr.common.util :refer [are3]]
+   [cmr.common.util :as common-util :refer [are3]]
```

Comment:
No need to modify this line.

Reply:
Right. I needed to use it before, but not anymore. Done.